### DISCUSSION TITLE
Help on making a custom view in the edit page

### DISCUSSION THREAD

**Discussion by nlvedwin** [filamentphp/filament#4913] [Help]
I am trying to load a custom view in the edit page. 

**Question is how i can send the current record in the custom view?**

**Question is there a way to use the the filament fields in a custom view? like i wanted to use rich editor for submitting comments?** 

_Thank you for advance! you're doing a great job on this project :)_ 

<img width="960" alt="image" src="https://user-images.githubusercontent.com/78135357/201460746-f2ddf212-d3bd-43be-bb3d-c0d2fa0c1684.png">

<img width="1331" alt="image" src="https://user-images.githubusercontent.com/78135357/201460723-bf8f9667-5332-487e-9729-d76be139a9b0.png">

**Comment by danharrin** [comment]
I would suggest embedding a separate Livewire component for the comments

so swap the ViewField for a View, and then inside the blade, render `@livewire('comments', ['record' => $getRecord()])`

then add a `public Model $record` to the comments component, which is the record
